### DISCUSSION TITLE
Local GitHub actions should work like a reusable workflow

### DISCUSSION THREAD

**Discussion by papadeltasierra** [community/community#37957] [Actions]
I have a number of jobs that all start...
`
   my_job:
      runs-on: ubuntu-latest
      steps:
        - uses: actions/checkout@v3
        - uses: actions/setup-node@v3
          with:
            node-version: '14'
        - uses: ./.github/actions/my_action
`
This is not DRY code as I cannot place the `checkout` and `setup-node` steps into my custom action because GitHub does not make the custom action available unless I checkout the code.
However, _some magic_ means that reusable_ workflows are available without checking out the code and I feel that actions should get the same special treatment.

**Comment by ossanna16** [comment]
Hi there @papadeltasierra and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)
